### DISCUSSION TITLE
How can I stream udp with ffmpeg using mediamtx?

### DISCUSSION THREAD

**Discussion by omergumus** [bluenviron/mediamtx#2028] [Questions]
ffmpeg -re -i https://input.m3u8 -vcodec copy -acodec aac -maxrate 1300k -bufsize 2500k -f rtsp "rtsp://192.168.1.2:8554/001?pkt_size=1316"

Since I installed and started the mediamtx service with a bash command like this, I can broadcast rtsp without any problems. My request is:
Can I do udp broadcast with ffmpeg using mediamtx udp protocol support without any problem?

When I run a command like the one below, the broadcast stops after a while (about 6 minutes). Or when watching on a set top box, the screen freezes and restarts every 6 minutes. It's as if the broadcast is renewing itself (while watching from the set top box, the screen goes black and the broadcast comes back as if I opened that channel again. Also in VLC, the broadcast freezes but does not stop. It seems like it continues to play but the screen image is frozen). But when I check my ffmpeg command I can see it still working. This only happens when I broadcast udp. There is no such problem when I broadcast http or rtsp. But I want to use udp protocol because I want to receive these broadcasts over more than one set top box in the same network. How can I fix this problem?

My request is:
Can I do udp broadcast with ffmpeg using mediamtx udp protocol support without any problem?

Note:
I tried almost all stable and static ffmpeg versions and couldn't solve the problem.

UDP stream command:
ffmpeg -re -i https://input.m3u8 -vcodec copy -acodec copy -maxrate 1300k -bufsize 2500k -f mpegts "udp://239.100.1.1:8002?pkt_size=1316"

My server:
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye

My server Ffmpeg version:ffmpeg version 5.1.1

**Comment by DJ10177** [comment]
Hi, I've come across the same issue where VLC is still playing the stream but the the picture is frozen for about 15 seconds. The same problem takes place when I stream using tcp. Have you managed to find a solution yet?

**Comment by aler9** [comment]
> When I run a command like the one below, the broadcast stops after a while (about 6 minutes). Or when watching on a set top box, the screen freezes and restarts every 6 minutes

open an issue and provide details in order to replicate the bug.

[a comment by an automated account is not shown]
